client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -61,25 +59,19 @@ const UnifiedDesignPickerStep: StepType< {
 		};
 	};
 } > = ( { navigation, flow, stepName } ) => {
-	// imageOptimizationExperimentAssignment, exerimentAssignment
 	const [ isLoadingExperiment, experimentAssignment ] = useExperiment(
 		'calypso_design_picker_image_optimization_202406'
 	);
 	const variantName = experimentAssignment?.variationName;
 	const oldHighResImageLoading = ! isLoadingExperiment && variantName === 'treatment';
 
-	const [ isGoalsAtFrontExperimentLoading, isGoalsAtFrontExperiment ] = useGoalsFirstExperiment();
```

Comment:
Both `isGoalsAtFrontExperimentLoading` and `isGoalsAtFrontExperiment` are always `false`